common/CommandControl.hpp

```diff
@@ -49,31 +53,51 @@ class LockManager
 
     static void setLockedUser(bool isLocked) { _isLockedUser = isLocked; }
     static void setHostReadOnly(bool isReadOnly) { _isHostReadOnly = isReadOnly; }
-
+    static void setTranslationPath(const std::string& lockedDialogLang);
     static std::string getUnlockTitle()
     {
-        return config::getString("feature_lock.unlock_title", "");
+        return config::getString(translationPath + ".unlock_title", "");
+    }
+    static std::string getUnlockLink()
+    {
+        return config::getString("feature_lock.unlock_link", "");
     }
-    static std::string getUnlockLink() { return config::getString("feature_lock.unlock_link", ""); }
     static std::string getUnlockDescription()
     {
-        return config::getString("feature_lock.unlock_description", "");
+        return config::getString(translationPath + ".unlock_description", "");
```

Comment:
@Rash419: I think instead of config::getString(translationPath + ".unlock_description", "");, I guess you want a function that checks if the particular "unlock_description" etc. exist in the given language.  I suspect it might happen there is an incomplete translation, and only eg. unlock_description is provided, but not the other - which I think will fail, and provide an empty string, instead of the default one (?)